kubernetes/helm_charts/core/nodebb/values.j2

```diff
@@ -24,4 +24,4 @@ strategy:
   maxunavailable: {{ nodebb_maxunavailable|default('25%') }}
 
 nodebb_url: {{ sunbird_nodebb_url }}
-redis_nodebb_host: {{ redis_nodebb_host }}
+mongo_nodebb_host: {{ mongo_nodebb_host }}
```

Comment:
what's the default value for this ?

Reply:
the default entry should be in all.yaml like
mongo_nodebb_host: group['mongo'] | join ','

Reply:
Updated

Reply:
will the application support ',' seperated ips ?